scaling_performance/scaling_cluster_monitoring.adoc

```diff
@@ -0,0 +1,131 @@
+[[scaling-performance-cluster-monitoring]]
+= Scaling Cluster Monitoring Operator
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+the link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]
+As an {product-title} administrator, you can view system resources, containers and components metrics in
+one dashboard interface (Grafana).
+
+This topic provides information on scaling the cluster monitoring operator.
+
+
+[[cluster-monitoring-recommendations-for-OCP]]
+== Recommendations for {product-title}
+* Use at least 3 infra nodes as best practice (for 2 Prometheus pods and operator pods)
+xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
+nodes].
+* Use at least 3 CNS nodes (with NVMe drives)
```

Comment:
@mrsiano @ahardin-rh I think CNS should be replaced with OCS - Openshift Container Storage - check naming https://www.redhat.com/de/technologies/cloud-computing/openshift-container-storage

Reply:
@ekuric Good catch. That change was introduced in 3.10: https://docs.openshift.com/container-platform/3.10/release_notes/ocp_3_10_release_notes.html#ocp-310-CNS-is-now-RHOCS